fcc-test-suite-template-master/index.html

```diff
@@ -60,6 +60,34 @@ <h1 id="title">Tito and Danny Bazar</h1>
          <button id="submit" type="submit" class="txts">Submit Feedback</button>
        </div>
      </form>
+     <ul class="projects">
+       <li class="project">
+         <img class="image" src="./img/snapshot.png" alt="">
+         <div class="projectdetleft">
+           <h3 class="projecttitle">Tonic</h3>
+           <ul class="projectdetails">
+             <li class="company">CANOPY</li>
+             <li class="role">Back End Dev</li>
+             <li class="time">2015</li>
+           </ul>
+
+           <p class="paraggraph">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups
+             required.</p>
+
+           <ul class="tools">
+             <li>Html</li>
+             <li>CSS</li>
+             <li>Javascript</li>
+           </ul>
+
```

Comment:
OPTIONAL

Instead of using a blank line, consider using `<br>` to add a line break.